frontend/add-leads/add-event-step.component.tsx

```diff
@@ -1,7 +1,229 @@
 import React from "react";
+import { useCss } from "kremling";
+import AddLeadStepHeader from "./add-lead-step-header.component";
+import imgSrc from "../../icons/148705-essential-collection/svg/map-location.svg";
+import dayjs from "dayjs";
+import easyFetch from "../util/easy-fetch";
 
 export default function AddEventStep(props: AddEventStepProps) {
-  return <div className="card">Add an event!</div>;
+  const scope = useCss(css);
+  const [isExistingEvent, setIsExistingEvent] = React.useState(true);
+  const [eventDate, setEventDate] = React.useState(
+    dayjs().format("YYYY-MM-DD")
+  );
+  const [allEvents, setAllEvents] = React.useState([]);
+  const [eventName, setEventName] = React.useState("");
+  const [eventLocation, setEventLocation] = React.useState("");
+  const [totalAttendance, setTotalAttendance] = React.useState(1);
```

Comment:
Could total attendance ever be 0?

Reply:
I don't think it makes sense to add an event that didn't have any attendance. Especially in the context of creating a lead that is associated with that event. If we have a lead that attended an event, there's at least one attendee